CI.hs

```diff
@@ -441,6 +446,7 @@ buildDists
     let ghcOpts = case ghcFlavor of Da {} -> ghcOptionsOpt ghcOptions;  _ -> ghcOptionsWithHaddock ghcOptions
     stack $ "--no-terminal --interleaved-output build " ++ ghcOpts ++ " ghc-lib-parser"
     stack $ "--no-terminal --interleaved-output build " ++ ghcOptionsOpt ghcOptions ++ " ghc-lib"
+#if 0
```

Comment:
is `ghc-lib-test-mini-hlint` broken on 9.6.1? if so, I'd say we should exclude just that version explicitly, rather than disabling it entirely

Reply:
oops! good catch! sorry, unintended change. i fixed and pushed. unfortunately, this will mean needing to `/azp run` again. sorry!

Reply:
argh. re-enabling has reintroduced what i tricked myself into believing was a solved problem. i'll have to work something out. putting this back in draft for now.

Reply:
thank you @shayne-fletcher!

Reply:
this problem would appear to be issue https://github.com/commercialhaskell/stack/issues/5866.